### PR TITLE
Update github templates

### PR DESCRIPTION
**Pull request by naosense**
Update some templates for github issues/pr/question
Related to #10

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/--question.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 name: "❓ Question"
-about: Please use https://discuss.akka.io for questions
+about: Please use https://discuss.pekko.io for questions
```
Comment (pjfanning) [inline .github/ISSUE_TEMPLATE/--question.md:3]: we don't have a discuss.pekko.io - could you use https://github.com/apache/incubator-pekko/discussions instead?

Comment (naosense) [reply .github/ISSUE_TEMPLATE/--question.md:3]: ok, I'll fix that

**Review by pjfanning — APPROVED** [review]

**Comment by pjfanning** [comment]
@naosense could I ask for another change?

Could you replace this (in 2 templates_?

```
Thanks, happy hakking!
```

```
Thanks for contributing!
```

hakking is based on 'Akka' - and we want to start moving away from that name

**Comment by naosense** [comment]
@pjfanning Yeah, happy to do it!

File: .github/ISSUE_TEMPLATE/---feature-request.md [hunk 2]
```diff
@@ -6,12 +6,12 @@ about: Suggest an idea for this project
 
 <!--
 Please report issues regarding specific projects in their respective issue trackers, e.g.:
- - Akka HTTP: https://github.com/akka/akka-http/issues
- - Alpakka:   https://github.com/akka/alpakka/issues
- - Akka Persistence Cassandra Plugin: https://github.com/akka/akka-persistence-cassandra/issues
+ - Pekko: https://github.com/apache/incubator-pekko/issues
```
Comment (pjfanning) [inline .github/ISSUE_TEMPLATE/---feature-request.md:9]: Could you replace (in 2 templates):
```
Pekko: https://github.com/apache/incubator-pekko/issues
```
with
```
Pekko Connectors: https://github.com/apache/incubator-pekko-connectors/issues
```

Pekko Connectors is the equivalent of Alpakka in the old template

**Comment by pjfanning** [comment]
thanks - merged

**Pull request merged**